task8/genericClass.ts

```diff
@@ -0,0 +1,50 @@
+abstract class Pet {
+    name: string;
+    type: string;
+
+    constructor(name: string, type: string) {
+        this.name = name;
+        this.type = type;
+    }
+   
+    toString(): string {
+        return `${this.name}(${this.type})`;
+    }
+
+}
+
+enum FoodType {
+    Meat,
+    Vegetarian,
+    Anything
```

Comment:
Let replace it, because 'anything' doesn't have any sense:
for example anything => RawFood